packages/cli/src/lib/pipelines/render-pipeline.ts

```diff
@@ -45,16 +46,16 @@ export default async function renderPipeline(
         const email = row.owner && row.owner.email
 
         if (email) {
-          return email.endsWith('@herokumanager.com') ? `${row.split('@')[0]} (team)` : email
+          return email.endsWith('@herokumanager.com') ? `${email.split('@')[0]} (team)` : email
         }
       },
     }
   }
 
-  const developmentApps = sortBy(pipelineApps.filter(app => app.coupling.stage === 'development'), ['name'])
-  const reviewApps = sortBy(pipelineApps.filter(app => app.coupling.stage === 'review'), ['name'])
-  const stagingApps = sortBy(pipelineApps.filter(app => app.coupling.stage === 'staging'), ['name'])
-  const productionApps = sortBy(pipelineApps.filter(app => app.coupling.stage === 'production'), ['name'])
+  const developmentApps = sortBy(pipelineApps.filter(app => app.pipelineCoupling.stage === 'development'), ['name'])
```

Comment:
Yes, we were using an object that had all App's attributes and adding a `coupling` set to a PipelineCoupling object. Because it's a custom type (not existent at the Platform API schema) I just changed semantics and called the attribute `pipelineCoupling` to make it easier to understand what goes in there, but that's all.

The `generation` I had to add it at another interface derived from PipelineCoupling, I called it `PipelineCouplingSdk`. My expectation is that we remove that when we have our Fir types generated from the SDK variant correctly. I didn't want to manually add that attribute to the `/lib/types/fir.d.ts` file because they could get overwritten if we generate and update that file again.